source/core/index.ts

```diff
@@ -684,6 +685,7 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 			assert.any([is.string, is.object, is.array, is.undefined], options.https.key);
 			assert.any([is.string, is.object, is.array, is.undefined], options.https.certificate);
 			assert.any([is.string, is.undefined], options.https.passphrase);
+			assert.any([is.string, is.array, is.undefined], options.https.pfx);
```

Comment:
Is there a way to assert that a Buffer is allowed? It doesn't seem captured by these other types?

Reply:
`is.buffer`? https://github.com/sindresorhus/is#buffervalue

Reply:
b039a37707d1d2774282205c290997493234b5b1